components/Vote/Descriptions.tsx

```diff
@@ -11,8 +14,10 @@ const Descriptions = () => {
       <p className="font-roboto text-2xl">
         None of the parties may perform legally relevant acts on the musical
         work without the written authorization of the{' '}
-        <span className="text-danger-dark font-rubik">51%</span> of the
-        ownership, such as but not limited to the following:
+        <span className="text-danger-dark font-rubik">
+          {votePercentage || '51'}%
```

Comment:
What do you think about having the default value of `votePercentage` be `51%` so we can remove this fallback? Also provides accurate default value for votePercentage across the `provider` usage right???

Reply:
Sure